mautic/mautic
# [UI] New import contacts screen

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR changes a bit of the importing screen

### Before:

![image](https://github.com/user-attachments/assets/e2886a53-b281-4654-b5f8-2f6a151b14b2)

![image](https://github.com/user-attachments/assets/60237d32-b28c-4680-a371-f5dfe8ffd157)

![image](https://github.com/user-attachments/assets/7cc334bc-faf4-4c7b-8bbe-ba83f986dabd)


### After

[Screencast From 2025-03-31 21-34-12.webm](https://github.com/user-attachments/assets/6b766dbe-83dd-4b24-854b-56f564ee0013)

![image](https://github.com/user-attachments/assets/cc0c65dc-b6e7-4668-964a-26cad82933b9)

![image](https://github.com/user-attachments/assets/e62d30a1-8d8a-4670-8d87-bb0a0a785c66)

![image](https://github.com/user-attachments/assets/f1ba0ae9-6e43-42ae-b862-5241370eae96)

![image](https://github.com/user-attachments/assets/2ec0a123-90fa-4cdb-b42e-0f2705f2fa52)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > More actions > Import > See the page and download the sample file
3. Attach the sample file and click Upload and match fields
4. Match the field company to Company Name (instead of Primary company)
5. Click to import and see
